src/views/pages/globalping/users.html

```diff
@@ -0,0 +1,448 @@
+<link rel="ractive" href="../../components/notification.html" name="c-notification">
+
+<r-page noYield="{{noYield}}" title="{{title}}" description="{{description}}">
+	<c-notification></c-notification>
+	<c-gp-header additionalClasses="header-with-gp-translucent-bg"></c-gp-header>
+
+	{{#if !@shared.isServer && !@shared.googleMapsLoaded && username}}
+		<script>
+			function onDocumentReady (fn) {
+				if (document.readyState !== 'loading') {
+					fn();
+				} else {
+					document.addEventListener('DOMContentLoaded', fn);
+				}
+			}
+			/* eslint-disable */
+			function initMap () {
+				onDocumentReady(function () {
+					app.loaded = true;
+
+					if (app.router.route && app.router.route.view && app.router.route.view.initMap) {
+						app.router.route.view.initMap();
+					}
+				});
+			}
+		</script>
+		<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBTICLY5LgWNJ0Uly61Q1sNXz_1KZIECwg&amp;language=en&amp;callback=initMap&libraries=marker&loading=async" async></script>
+		<script src="https://cdn.jsdelivr.net/npm/@googlemaps/markerclusterer@2.5.3/dist/index.umd.min.js"></script>
+	{{/if}}
+
+	<div class="p-users">
+		<div class="please-wait-sign {{#if username.length}} not-displayed{{/if}}">
+			Please wait while user data loads.
+		</div>
+
+		<div class="page-content {{#unless username.length}} not-displayed{{/unless}}">
+			<div class="page-content_side-menu">
+				<div class="menu-section">
+					<div class="section-list">
+						<div>
+							{{username}}
+						</div>
+					</div>
+				</div>
+			</div>
+
+			<div class="page-content_main">
+				<div class="page-content_main_section">
+					<div class="p-users_map-wrapper">
+						<div id="gp-map" class="p-users_map-wrapper_map"></div>
+					</div>
+				</div>
+
+				<div class="page-content_main_section">
+					<h2 class="page-content_main_section_title">List of probes</h2>
+					{{#each userProbes}}
+						<div class="page-content_main_section_probe">
+							<div class="page-content_main_section_probe_details">
+								<div class="page-content_main_section_probe_details_icon">
+									<img width="24"
+									height="24"
+									src="{{@shared.assetsHost}}/img/globalping/globalping-probe.svg">
+								</div>
+								<div class="page-content_main_section_probe_details_info">
+									<div class="page-content_main_section_probe_details_info_city">
+										<img width="16.8" height="12" src="https://cdn.jsdelivr.net/npm/country-flag-icons@1.5.5/3x2/{{location.country}}.svg">
+										<span class="page-content_main_section_probe_details_info_city_name">
+											{{location.city}}, {{location.country}}
+										</span>
+									</div>
+									<span class="page-content_main_section_probe_details_info_location">
+										{{location.network}}
+									</span>
+								</div>
+							</div>
+							<div class="page-content_main_section_probe_tag">
+								{{#each tags}}
+									<span>
+										{{this}}
+									</span>
+								{{/each}}
```

Comment:
For tags, please:
 - don't force two lines if not needed; if they fit in one line, they should be next to each other,
 - try to replicate [this](https://github.com/jsdelivr/globalping-dash/blob/9ad04759b6f07c4ccbf56ccc1580011353cd10bc/pages/probes.vue#L303-L311) to calculate the number of tags that fits in two rows, and if there are more, hide the rest behind a tooltip,
 - try to add `text-overflow: ellipsis` to handle very long values.

Here's how it works in the dashboard: https://dl.dropboxusercontent.com/scl/fi/x22lmik9xmbxs2pp0z20d/2025-03-17_15-00-29.gif?rlkey=fcwpqupl17p3og3inxltipo5w&dl=0

Reply:
I was working on this and got reminded by your comment; I can't find your username in the probes' tags anywhere. I can see in the video you shared you have some tags available but I can't find it. Am I using the correct endpoint?

Here's the list of unique tags sorted alphabetically:

![image](https://github.com/user-attachments/assets/8b226cb2-0323-46af-8a94-d07cc40cde00)

Reply:
Yeah I had previously checked by clearing the storage but didn't clear the storage when I shared the screenshot I'm sorry. Your tags where there after cleanup.

Used `as-tooltip` to show the tip for every tag (in case a long tag was truncated, might have been better to use `title` there but for the sake of consistency...) and enabled tooltip on overflow counter.